### PR TITLE
Expose priority as an API for Composer

### PR DESCRIPTION
**Pull request by mbarton**
https://github.com/guardian/flexible-content/pull/3359

Also added bonus logging of errors when getting content since it was obstructing me debugging why my local build was running (I had the wrong port in my config)

### CODE REVIEW

File: app/controllers/Api.scala [hunk 1]
```diff
@@ -57,8 +57,12 @@ object Api extends Controller with PanDomainAuthActions with SharedSecretAuth {
     val qs: Map[String, Seq[String]] = queryString(req)
 
     CommonAPI.getStubs(qs).asFuture.map {
-      case Left(_) => InternalServerError
-      case Right(contentResponse) => Ok(contentResponse.asJson.noSpaces)
+      case Left(err) =>
```
Comment (sihil) [inline app/controllers/Api.scala]: Without opening an IDE I can't tell if this is an exception. If so then it should be in the second param in the logger line. I'm guessing it's a string though.

File: app/controllers/Api.scala [hunk 2]
```diff
@@ -282,6 +286,12 @@ object Api extends Controller with PanDomainAuthActions with SharedSecretAuth {
     }
   }
 
+  def priorities = CORSable(atomCorsAble) {
+    AuthAction {
```
Comment (sihil) [inline app/controllers/Api.scala]: Should this be an `APIAuthAction`?

File: app/controllers/Api.scala (hunk 1 above)
Comment (mbarton) [reply app/controllers/Api.scala]: It's an [ApiError](https://github.com/guardian/workflow-frontend/blob/master/app/lib/Response.scala#L8) wrapper object around the HTTP response from the Workflow datastore app

File: app/controllers/Api.scala (hunk 2 above)
Comment (mbarton) [reply app/controllers/Api.scala]: Agreed, all the metadata endpoints should be (`statuses`, `sections`, `allowedAtomTypes` and `priorities`).

I've raised https://github.com/guardian/workflow-frontend/pull/169 for it

**Pull request merged**